doc/user/content/sql/create-source/kafka.md

```diff
@@ -298,6 +298,67 @@ As long as any offset continues increasing, Materialize is consuming data from
 the upstream Kafka broker. For more details on monitoring source ingestion
 progress and debugging related issues, see [Troubleshooting](/ops/troubleshooting/).
 
+### Committing offsets
+
+Kafka sources commit offsets when the messages up through that offset have been
+durably recorded in Materialize's storage layer. This enables Kafka tools that
+use committed offsets to determine consumer lag to monitor Materialize's
+progress through the source's topic.
+
+Prefer using Materialize's native [progress
+monitoring](#monitoring-source-progress) to the committed offsets in Kafka. The
+progress information in Materialize will be more up to date than the committed
+offsets in Kafka.
```

Comment:
```suggestion
To support Kafka tools that monitor consumer lag, Kafka sources commit offsets
once durably recording them in Materialize's storage layer.

However, rather than relying on committed offsets, Materialize suggests using
our native [progress monitoring](#monitoring-source-progress) which always
contains more up-to-date data.
```

Reply:
Thanks, accepted with a tiny modification.